examples/rich-text-presence/client.js

```diff
@@ -36,9 +36,12 @@ doc.subscribe(function(err) {
 });
 
 function initialiseQuill(doc) {
-  var quill = new Quill('#editor', {
-    theme: 'bubble',
-    modules: {cursors: true}
+    var quill = new Quill('#editor', {
+    theme: 'snow',
```

Comment:
Any reason in particular for changing the theme?

Reply:
The theme has a toolbar.  It is easier to test behavior cursor and the toolbar interactions with it.

Reply:
🤷🏼‍♂️ I guess it doesn't matter which theme we use, but if you need to do any particularly in-depth testing with cursors and toolbar interaction, that should really be a concern of `quill-cursors`.